templates/node.js-functionapp-to-azure-on-linux.yml

```diff
@@ -1,5 +1,5 @@
-# Node.js Function App on Linux
-# Build and deploy a Node.js Function app on Linux.
+# Node.js Function App to Azure on Linux
+# Build a Node.js Function app and deploy it to Azure on Linux.

```

Comment:
deploy it to Azure as a Linux function app

Reply:
done